.github/workflows/security-scan.yml

```
@@ -0,0 +1,218 @@
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Setup dependencies
+        run: |
+          npm ci
+          sudo curl -SL https://github.com/docker/compose/releases/download/v2.10.2/docker-compose-linux-x86_64 -o /usr/local/bin/docker-compose
+          sudo chmod +x /usr/local/bin/docker-compose
+          docker pull ${DOCKER_REPO}:${DOCKER_APPLICATION_TAG}
+
+      - name: Run dynamic security scan
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:2:38: Double quote to prevent globbing and word splitting

(shellcheck)